tools/service-account-provider/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM golang:1.16-buster as build
+
+WORKDIR /go/src/app
+ADD . /go/src/app
+
+RUN go build -o /go/bin/app main.go
+
+FROM gcr.io/distroless/base-debian10
+COPY --from=build /etc/ssl/certs /etc/ssl/certs
+COPY --from=build /go/bin/app /
+
+CMD ["/app"] 
```

Comment:
Please use a multi-stage docker file ? https://docs.docker.com/develop/develop-images/multistage-build/

Essentially build using the golang image but the final image should not contain the compiler, you can use Alpine or Debian

Reply:
I am sorry, I thought I am using a multi-stage. Can you take another look, I am not sure what I am missing.